csrankings-k.csv

```diff
@@ -624,6 +624,7 @@ Khalid Mahmood Malik,Oakland University,https://oakland.edu/secs/directory/mahmo
 Khalil Sima'an,University of Amsterdam,https://staff.fnwi.uva.nl/k.simaan,NOSCHOLARPAGE
 Khin Than Win,University of Wollongong,https://scholars.uow.edu.au/khin-win,tM8rvvQAAAAJ
 Khoa Nguyen 0002,University of Wollongong,https://scholars.uow.edu.au/khoa-nguyen,RQ9tJVIAAAAJ
+Khuong Nguyen,Royal Holloway University of London,https://khuong.uk,XVvcUCEAAAAJ

```

Comment:
Needs entry for alias as well: https://dblp.org/pid/118/9856.html

Reply:
Added the alias; however, the alias commit validation failed, stating "Possibly invalid name (Khuong An Nguyen). This may be a disambiguation entry", although this is somewhat the expected result.